java-checks-test-sources/default/pom.xml

```diff
@@ -921,25 +927,25 @@
       <artifactId>aspectjweaver</artifactId>
       <version>1.9.9.1</version>
       <scope>provided</scope>
-  </dependency>
-  <dependency>
-    <groupId>io.micronaut</groupId>
-    <artifactId>micronaut-http</artifactId>
-    <version>3.7.2</version>
-    <scope>provided</scope>
-	</dependency>
+    </dependency>
```

Comment:
Let's avoid unrelated reformatting of the file. This also applies to line 612 and the 944-948. The reason is the it changes git blame layer and makes it more difficult to understand what was done in a PR.